backend/DAO/employerDAO.js

```diff
@@ -3,17 +3,17 @@ const database = require('../DB/connection');
 module.exports = {
     getEmployer: async (employer_id) => {
         let result = await database('employer')
-           .select('name', 'location', 'urllogo')
+            .select('name', 'location', 'urllogo')
```

Comment:
We should delete the urllogo (also in the next sprint if you don't have time now)